sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1120,11 +1129,34 @@ def test_datapath_state(self, api, channel, states):
 
         return done
 
+    def get_host_tx_status(self, lport):
+        host_tx_ready = 'False'
+
+        asic_index = self.port_mapping.get_asic_id_for_logical_port(lport)
+        state_port_tbl = self.xcvr_table_helper.get_state_port_tbl(asic_index)
+
+        found, port_info = state_port_tbl.get(lport)
+        if found and 'host_tx_ready' in dict(port_info):
+            host_tx_ready = dict(port_info)['host_tx_ready']
+        return host_tx_ready
+
+    def get_port_admin_status(self, lport):
+        admin_status = 'down'
+
+        asic_index = self.port_mapping.get_asic_id_for_logical_port(lport)
+        cfg_port_tbl = self.xcvr_table_helper.get_cfg_port_tbl(asic_index)
+
+        found, port_info = cfg_port_tbl.get(lport)
+        if found:
+            # Check admin_status too ...just in case
+            admin_status = dict(port_info)['admin_status']
+        return admin_status
+
     def task_worker(self):
-        self.log_notice("Starting...")
+        self.log_notice("Starting CmisTaskMgr with pid:{}".format(self.task_process.pid))
```

Comment:
If you checked the syslog of pmon#xcvrd, the pid is available over there, on the other hand, we already have the 'CMIS' prefix in the CmisManagerTask.log_notice(), hence this is not necessary